views/layouts/MainLayout/index.jsx

```diff
@@ -53,7 +53,7 @@ const MainLayout = () => {
     <div className="MainLayout_container">
       <Sidebar sidebarOpen={true} user={user} onLogout={handleLogout} />
 
-      <main className="MainLayout_main">
+      <div className="MainLayout_main">
```

Comment:
why change this?

Reply:
the main tag goes behind the sidebar, but the div gets positioned adjacent to the sidebar when it's position is static, I verified the participants table and it works there too. This way we don't have to use a margin left.

Reply:
we don't want to use position static because the sidebar will scroll with the page. Either way, main vs div shouldn't make a difference.

Reply:
[a8fa89f](https://github.com/AMP-SCZ/dpdash/pull/478/commits/a8fa89f3d10b6e4db122a9aa52573c428d3682dc)